### DISCUSSION TITLE
Admin Console take 15 mins to load every time

### DISCUSSION THREAD

**Discussion by Msquared63** [keycloak/keycloak#19322] [Q&A]
Hi,

I upgraded from V12 to V21.  I have a v12 posgres backend.  I have only 479 Realms.

Every time I log into the Admin console it take 15 mins or more to load the page.  With tcpdump I can see it's reloading from the database instead of the cache.

No clue to what can be wrong?

Thanks

Mike

**Comment by ssilvert** [comment]
Please file an issue.

https://github.com/keycloak/keycloak/issues
